src/razor/src/dynamicFile/dynamicFileInfoHandler.ts

```diff
@@ -40,7 +40,7 @@ export class DynamicFileInfoHandler {
             }
         );
         this.documentManager.onChange(async (e) => {
-            if (e.kind == RazorDocumentChangeKind.csharpChanged && !e.document.isOpen) {
+            if (e.kind == RazorDocumentChangeKind.csharpChanged && !e.document.isOpen && e.changes.length > 0) {
```

Comment:
this is the change that reduces updates. Even if our version is bumped as long as the csharp text didn't change no point in sending a notification to Roslyn